src/main/resources/db/migration/V12__opprettet_tid.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE iverksett
+    ADD COLUMN opprettet_tid TIMESTAMP(3) NOT NULL DEFAULT LOCALTIMESTAMP;
+ALTER TABLE brev
+    ADD COLUMN opprettet_tid TIMESTAMP(3) NOT NULL DEFAULT LOCALTIMESTAMP;
+ALTER TABLE behandlingsstatistikk
+    ADD COLUMN opprettet_tid TIMESTAMP(3) NOT NULL DEFAULT LOCALTIMESTAMP;
+
+ALTER TABLE iverksett_resultat
+    ADD COLUMN opprettet_tid TIMESTAMP(3) NOT NULL DEFAULT LOCALTIMESTAMP;
+ALTER TABLE iverksett_resultat
+    ADD COLUMN endret_tid TIMESTAMP(3) NOT NULL DEFAULT LOCALTIMESTAMP;
```

Comment:
Må gjøre noe smartare her. Kan ikke sette default localtimestamp for allerede opprettede entiteter her 🤦

Reply:
Denne er vrien, ja. Må isåfall få satt tid basert på noe - iverksett har jo vedtaksresultat - så KAN jo oppdatere basert på dette..?